x/sequencer/keeper/sequencer.go

```diff
@@ -88,6 +89,21 @@ func (k Keeper) GetSequencer(ctx sdk.Context, sequencerAddress string) (val type
 	return val, true
 }
 
+func (k Keeper) GetSequencerByRollappByStatus(ctx sdk.Context, rollappId, seqAddress string, status types.OperatingStatus) (val types.Sequencer, found bool) {
```

Comment:
not sure why u can't use `GetSequencersByRollappByStatus` in the test file that uses it